core/primitives/transaction_validity.hpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_TRANSACTION_VALIDITY_HPP
+#define KAGOME_CORE_PRIMITIVES_TRANSACTION_VALIDITY_HPP
+
+#include <cstdint>
+#include <variant>
+#include <vector>
+
+namespace kagome::primitives {
+
+  /**
+   * This is the same structure as in
+   * https://github.com/paritytech/substrate/blob/master/core/sr-primitives/src/transaction_validity.rs
+   */
+
+  /// Priority for a transaction. Additive. Higher is better.
+  using TransactionPriority = uint64_t;
+
+  /**
+   * Minimum number of blocks a transaction will remain valid for.
+   * `TransactionLongevity::max_value()` means "forever".
+   */
+  using TransactionLongevity = uint64_t;
+
+  /**
+   * Tag for a transaction. No two transactions with the same tag should
+   * be placed on-chain.
+   */
+  using TransactionTag = std::vector<uint8_t>;
+
+  struct Valid {
```

Comment:
Maybe it would be better to call it a noun? I don't see a comment above it so it's a little confusing valid what it is.

Reply:
Just to be consistent with substrate interface. I will add short comment,but I want to keep the same naming as [here ](https://github.com/paritytech/substrate/blob/master/core/sr-primitives/src/transaction_validity.rs#L38)